pinot-common/src/test/java/com/linkedin/pinot/common/utils/SegmentNameBuilderTest.java

```diff
@@ -35,14 +35,14 @@ public void testSegmentNameBuilder() {
     assertEquals(shortV1Name, "myTable_REALTIME_1234567_0__ALL__1234567");
     assertEquals(v2Name, "myTable_REALTIME__0__1__20160609T2142Z");
 
-    // Check v1/v2 format detection
-    assertEquals(isRealtimeV1Name(oldV1Name), true);
-    assertEquals(isRealtimeV1Name(shortV1Name), true);
-    assertEquals(isRealtimeV1Name(v2Name), false);
+    // Check high level consumer/low level consumer format detection
+    assertEquals(isHighLevelRealtimeSegmentName(oldV1Name), true);
+    assertEquals(isHighLevelRealtimeSegmentName(shortV1Name), true);
+    assertEquals(isHighLevelRealtimeSegmentName(v2Name), false);
 
-    assertEquals(isRealtimeV2Name(oldV1Name), false);
-    assertEquals(isRealtimeV2Name(shortV1Name), false);
-    assertEquals(isRealtimeV2Name(v2Name), true);
```

Comment:
Can you please change segment name  to exclude "REALTIME" so that it serves as an example for the same naming scheme that we plan to use for the low-level consumer segments?